.azure-pipelines/azure-pipelines-job-groups.yml

```diff
@@ -16,20 +19,25 @@ parameters:
   default: ''
 - name: 'timeoutInMinutes'
   type: 'number'
-  default: 1440
+  default: 600
+- name: 'pool'
```

Comment:
Can be removed, it supported by design.

Reply:
fix

---
In reply to: [606133050](https://github.com/Azure/sonic-buildimage/pull/7198#discussion_r606133050) [](ancestors = 606133050)